.drone/drone.jsonnet

```diff
@@ -497,40 +497,58 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
 };
 
 [
-  pipeline('loki-build-image') {
-    local build_image_tag = '0.31.2',
+  pipeline('loki-build-image-' + arch) + arch_image(arch) {
+    local build_image_tag = '0.32.0-test-' + arch,
     workspace: {
       base: '/src',
       path: 'loki',
     },
     steps: [
       {
-        name: 'test-image',
+        name: 'push-image',
         image: 'plugins/docker',
         when: onPRs + onPath('loki-build-image/**'),
+        environment: {
+          DOCKER_BUILDKIT: 1,
+        },
         settings: {
           repo: 'grafana/loki-build-image',
           context: 'loki-build-image',
           dockerfile: 'loki-build-image/Dockerfile',
+          username: { from_secret: docker_username_secret.name },
+          password: { from_secret: docker_password_secret.name },
           tags: [build_image_tag],
-          dry_run: true,
+          dry_run: false,
+          platform: 'linux/' + arch,
         },
       },
+    ],
+  }
+  for arch in ['amd64', 'arm64']
+] + [
+  pipeline('loki-build-image-publish') {
+    local build_image_tag = '0.32.0-test',
```

Comment:
Is this still needed? It doesn't seem to be used anymore.

Reply:
You are right. I need to check if this is a bug.